app/script/conversation/ConversationRepository.js

```diff
@@ -1144,8 +1144,8 @@ z.conversation.ConversationRepository = class ConversationRepository {
       .then(response => {
         const event = response ? response.event : undefined;
         if (event) {
-          this.eventRepository.injectEvent(response.event, z.event.EventRepository.SOURCE.BACKEND_RESPONSE);
           this.logger.debug(`Successfully added bot to conversation '${conversationEntity.display_name()}'`, response);
+          return this.eventRepository.injectEvent(response.event, z.event.EventRepository.SOURCE.BACKEND_RESPONSE);
```

Comment:
Does `injectEvent` return the event JSON as well? If not we should make sure we do so here in order to not change the return value of the function.

Reply:
yes